root/frontend/src/components/MinimalHeader/MinimalHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import useStyles from './MinimaHeader.style';
+
+function MinimalHeader() {
+  const classes = useStyles();
+  return (
+    <div className={classes.header}>
+      <div className={classes.logo} />
+      <nav className={classes.nav}>
+        <NavLink className={({ isActive }) => (isActive ? classes.activeLink : classes.links)} to="/">
```

Comment:
Is there somewhere that `isActive` is defined?

If not and it's just placeholder, I believe `react-router-dom` has the ability to get the current location with `useLocation()`. We may be able to utilize that here to check the path instead of having it passed in as a prop (at least that is what I assumed this setup was for).

Reply:
`isActive` is built-in when using `NavLink` which helps determine which link we are currently on